Makefile

```diff
@@ -116,3 +116,8 @@ dev-compose-down-linux:
 
 profile:
 	pipenv run python profile_application.py
+
+generate-integration-test:
+	docker-compose -f docker-compose-dev-mac.yml up -d
```

Comment:
Do we need this? If so, this should be a prerequisite in the readme for running the script, as it falls over now if it's already running?

Reply:
Removed now, added something about it in the readme.